### ISSUE TITLE
NES / SNES Classic [Which to buy?]

### ISSUE THREAD

**Issue by Dorzalty** [ClusterM/hakchi2#1410]
Hello,

I'm really confused on which version of these consoles to buy. Do I need to buy the NES / SNES MINIS which apparently are **only** European (I live in the U.S) or does the hakchi2 mod also work on the NES / SNES Editions like this:

https://www.nintendo.com/super-nes-classic/
https://www.nintendo.com/nes-classic/
(I don't see an option for Minius for the U.S)

If I need to buy the European versions, is there anything else I need to know, like do I need to purchase anything else to be able to use it within the U.S?

Thanks~

**Comment by trcjr** [comment]
The current generation of NES mini and SNES mini are more or less the same. I picked up an NES mini from Best Buy and it worked with hakchi2 without issue.

**Comment by Dorzalty** [comment]
> 
> 
> The current generation of NES mini and SNES mini are more or less the same. I picked up an NES mini from Best Buy and it worked with hakchi2 without issue.

Thank you so much for the reply! I just made my purchase. I can't wait to get it up and running! :+1:

**Comment by Nubion** [comment]
I think the SNes Mini is better, because there are two controllers with it. In addition, the controller cable is almost twice as long as that of the NES Mini. The hardware of both devices is almost the same. Therefore, you should pay more attention to the accessories.

**Comment by Dorzalty** [comment]
That's true. I was intending to buy to play with family, so I bought the one with more controllers. It arrived today and I flashed it in less than five minutes. It was so incredibly easy and I was able to backup my entire SNES collection. Absolutely incredible!

**Comment by kaltinril** [comment]
Per the [FAQ](https://github.com/ClusterM/hakchi2/wiki/FAQ), the NES Mini has more writable partition space.

- **Q:** How many games can be uploaded to NES/SNES Mini?
- **A:** Writable partition of NES Mini is 384 megabytes in size, writable partition of SNES Mini is 290MB.
